### PR TITLE
CP-2879 Changed regex for news links assertion

### PR DESCRIPTION
**Pull request by tomaszmaleckimudita**
JIRA Reference: [CP-2879]

### :memo: Description ️
-  Changed regex for news links assertion according to decision from task
- 
### :exclamation: Checklist before merging a pull request

- change went through the QA process

[CP-2879]: https://appnroll.atlassian.net/browse/CP-2879?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/mudita-center-e2e/src/specs/news/news-check-online.e2e.ts [hunk 1]
```diff
@@ -50,10 +50,9 @@ describe("News Page Check", () => {
     await expect(newsCardElements).toHaveLength(6)
 
     for (let newsCard of newsCardElements) {
-      console.log(await newsCard.getText())
```
[the remark this answers is not in the document]
Comment (dkarski) [reply apps/mudita-center-e2e/src/specs/news/news-check-online.e2e.ts:53]: 47 line

**Review by lkowalczyk87 — APPROVED** [review]

File: apps/mudita-center-e2e/src/specs/news/news-check-online.e2e.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomaszmaleckimudita) [reply apps/mudita-center-e2e/src/specs/news/news-check-online.e2e.ts:53]: https://github.com/mudita/mudita-center/pull/1901/commits/737137988271c222b8c20fe28d5cc23ffea02aea

File: apps/mudita-center-e2e/src/specs/news/news-check-online.e2e.ts [hunk 2]
```diff
@@ -44,16 +44,14 @@ describe("News Page Check", () => {
     await lastNewsCardCommunityLinkText.scrollIntoView()
     await expect(lastNewsCardCommunityLinkText).toBeDisplayedInViewport()
   })
-  // TODO: to fix
   it.skip("Verify News Cards", async () => {
```
Comment (dkarski) [inline apps/mudita-center-e2e/src/specs/news/news-check-online.e2e.ts:47]:
```suggestion
  it("Verify News Cards", async () => {
```

Comment (tomaszmaleckimudita) [reply apps/mudita-center-e2e/src/specs/news/news-check-online.e2e.ts]: I didn't notice it sorry [42cddc3](https://github.com/mudita/mudita-center/pull/1901/commits/42cddc3e5c2943b748cec05709d0fbc14968748c)

**Review by dkarski — COMMENTED** [review]
please make sure it passes the test you fixed 😉

File: apps/mudita-center-e2e/src/specs/help/help-contact-support.ts [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    //     },
+    //   }
+    // )
+
+    await sendButton.click()
+    await sleep(3000)
+
+    expect(strictResponseMock.calls.length).toBe(1)
+  })
+})
```
Comment (dkarski) [inline apps/mudita-center-e2e/src/specs/help/help-contact-support.ts:150]: Are you sure that it should be a part of the PR?

**Review by dkarski — APPROVED** [review]

File: apps/mudita-center-e2e/src/specs/help/help-contact-support.ts (hunk 3 above)
Comment (tomaszmaleckimudita) [reply apps/mudita-center-e2e/src/specs/help/help-contact-support.ts:150]: Thanks for asking. No thats wrong. I will prepare new branch and PR instead

**Pull request closed without merging**